src/fdcache_entity.cpp

```diff
@@ -1745,7 +1745,7 @@ bool FdEntity::MergeOrgMeta(headers_t& updatemeta)
     if(0 <= atime.tv_sec){
         SetAtime(atime, true);
     }
-    is_meta_pending |= IsUploading(true);
+    is_meta_pending |= (IsUploading(true) || pagelist.IsModified());
```

Comment:
Is there any test we can add for this condition?  @CarstenGrohmann

Reply:
@gual This test case can be achieved by adding a simple test.
I will try to add it.